src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,42 @@
+package homework_2.pyramid_printer;
+
+import java.util.Scanner;
+
+public class PyramidPrinter {
+
+    public void run(){
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Opt: you should open scanner in try block. What if you'll get an exception in sout?

Reply:
Done.